samplefuncs.py

```diff
@@ -9,6 +9,16 @@
     import simplejson as json
 
 def OwnTracksTopic2Data(topic):
+    if type(topic) == unicode:
```

Comment:
why is this required, and can you not merge this if together with the one on line 22?

Reply:
Because in my situation, topic was not type of string but unicode. I have simply checked this.
Im afraid i dont understand this well.